extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -309,6 +316,58 @@ class AmpLightbox extends AMP.BaseElement {
   getHistory_() {
     return Services.historyForDoc(this.getAmpDoc());
   }
+
+  /**
+   * Sets the document body to transparent to allow for frame "merging" if the
+   * element is under FIE.
+   * The module-level execution of setTransparentBody() only works on inabox,
+   * so we need to perform the check on element build time as well.
+   * @private
+   */
+  maybeSetTransparentBody_() {
+    if (this.getAmpDoc().win != this.win) { // in FIE
+      setTransparentBody(this.getAmpDoc().win, /** @type {!HTMLBodyElement} */ (
+          dev().assert(this.win.document.body)));
+    }
+  }
+}
+
+
+/**
+ * Sets the document body to transparent to allow for frame "merging".
+ * @param {!Window} win
+ * @param {!HTMLBodyElement} body
+ * @private
+ */
+function setTransparentBody(win, body) {
+  Services.vsyncFor(win).run({
+    measure(state) {
+      state.alreadyTransparent =
+          computedStyle(win, body)['background-color'] == 'rgba(0, 0, 0, 0)';
+    },
+    mutate(state) {
+      if (!state.alreadyTransparent &&
+          getMode().development && !getMode().test) {
```

Comment:
Only `warn()` will be compiled out.

See https://github.com/ampproject/amphtml/blob/master/build-system/runner/src/org/ampproject/AmpCommandLineRunner.java#L54

Removing check and leaving only for `test`.